addons/sourcemod/scripting/shavit-tas.sp

```diff
@@ -0,0 +1,816 @@
+#pragma semicolon 1
+
+#include <sourcemod>
+#include <cstrike>
+#include <sdktools>
+#include <shavit>
+#include <smlib/entities>
+
+#pragma newdecls required
+
+public Plugin myinfo = 
+{
+	name = "TAS Style",
+	author = "Charles_(hypnos)",
+	description = "TAS Style",
+	version = "1.9.2",
+	url = "https://hyps.dev/"
+}
+
+#define RUN 0
+#define PAUSED 1
+#define BACKWARD 2
+#define FORWARD 3
+
+#define AutoStrafeTrigger 1
+
+int gi_Status[MAXPLAYERS+1];
```

Comment:
Global variables should follow the same pattern as the rest of the timer plugins